content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
+---
+
+# Quickstart
+
+This guide shows how to download, setup, and test a simple single-member cluster of etcd.
+
+## Before you start
+
+Before you begin, this etcd guide requires the following:
+
+* Linux or macOS
+
+## Get started
+
+## Step 1 - Download etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER=v3.4.15
+
+# choose either URL
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+
+DOWNLOAD_URL=${GITHUB_URL}
+```
+
+### Download
+
+On Linux
+```
+rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1
+```
+
+On macOS
+```
+rm -f /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-darwin-amd64.zip -o /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+unzip /tmp/etcd-${ETCD_VER}-darwin-amd64.zip -d /tmp
+mv /tmp/etcd-${ETCD_VER}-darwin-amd64/* /tmp/test-etcd
+```
+
+### Check the binaries
+
+Check that both etcd and etcdctl are correct.
+<!-- TODO: find out if there is a checksum we can use to verify the downloads -->
+
+```
+/tmp/test-etcd/etcd --version
+ETCDCTL_API=3 /tmp/test-etcd/etcdctl version
+```
+
+## Step 2 - Move etcd
+
+etcd can be run from any directory, so we can move the binary to a location in the system PATH.
+
+```
+mv /tmp/test-etcd/etcd /usr/local/bin/
+mv /tmp/test-etcd/etcdctl /usr/local/bin/
+```
```

Comment:
The move isn't essential to a quick start / getting started page. I'd drop this too.